jobs/async-upload/samples/sample_job_s3_to_oci.yaml

```diff
@@ -64,9 +67,13 @@ spec:
             - name: MODEL_SYNC_DESTINATION_TYPE
               value: "oci"
             - name: MODEL_SYNC_DESTINATION_OCI_URI
-              value: "oci://distribution-registry-test-service.default.svc.cluster.local:5001/my-model:latest"
+              value: "distribution-registry-test-service.default.svc.cluster.local:5001/my-model:latest"
+            - name: MODEL_SYNC_DESTINATION_OCI_REGISTRY
+              value: "distribution-registry-test-service.default.svc.cluster.local:5001"
             - name: MODEL_SYNC_DESTINATION_OCI_CREDENTIALS_PATH
-              value: "/opt/creds/destination"
+              value: "/opt/creds/destination/.dockerconfigjson"
+            - name: MODEL_SYNC_DESTINATION_OCI_BASE_IMAGE
+              value: "alpine:latest"
```

Comment:
I don't think we ever tested `alpine:latest`, I would recommend `busybox` which is the default also in KServe documentation: https://kserve.github.io/website/latest/modelserving/storage/oci/#prepare-an-oci-image-with-model-data:~:text=by%20the%20runtime%3A-,FROM%20busybox,-RUN%20mkdir%20/models